engine.py

```diff
@@ -28,16 +28,20 @@ class ShotgunEngine(Engine):
     """
         
     def __init__(self, *args, **kwargs):
-        # passthrough so we can init stuff
+        """
+        Constructor
+        """
+        # pass-through so we can init stuff
```

Comment:
I know this was already in here, but what is this comment trying to convey, exactly? Seems like it could be removed.